neo/SmartContract/ScriptFile.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Neo.IO;
+using System;
+using System.IO;
+
+namespace Neo.SmartContract
+{
+    public class ScriptFile : ISerializable
+    {
+        public enum ScriptEngine : int
+        {
+            NeoVM = 0x4E454F00 // NEO\0
```

Comment:
```
NEF3   //NEO Executable Format 3
```

Reply:
Not `NVM3` Erik?

Reply:
I think it's an executable file. So we call it `NEO Executable Format`.